storage/keyring_test.go

```diff
@@ -190,6 +190,91 @@ func (suite *KeyringSuite) TestRoleCredentials() {
+		AccessKeyId: "some not-so-secret-string",
+		SecretAccessKey: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer sollicitudin ligula ac lectus lobortis, sit amet laoreet lacus finibus. Fusce erat nibh, dignissim nec pretium et, rhoncus in felis. Ut maximus id risus id sagittis. Aenean ultrices ipsum vitae quam varius posuere. Nam elit nibh, dapibus et quam ut, vehicula bibendum nunc. Sed id enim quis justo congue fermentum. Aenean vitae nisl velit. Cras fringilla arcu ac quam volutpat, quis pharetra nibh gravida. In porttitor, lorem eu aliquet tristique, arcu ante facilisis sapien, at porttitor eros magna a magna. Nullam sit amet ornare mi. Suspendisse volutpat sollicitudin ligula, id accumsan lorem dictum sit amet. Sed tincidunt sapien et dui placerat, sed pulvinar ante maximus. Donec suscipit malesuada este
+		Nam viverra et urna non semper. Suspendisse scelerisque massa arcu, vitae congue arcu elementum sit amet. Ut nec sagittis odio, sed convallis elit. Aenean nec massa dapibus, hendrerit felis id, ultrices arcu. Sed finibus justo a erat rhoncus molestie. Sed faucibus dictum pharetra. Suspendisse suscipit felis vitae eros placerat scelerisque. Pellentesque vel tellus tellus.
+		Vivamus a ex quis massa porta condimentum tempus ac neque. Praesent lacus risus, ultricies nec convallis id, sollicitudin non diam. Nullam tempus velit quis neque pretium, ut sollicitudin nisl sodales. Duis elementum, magna a maximus sagittis, metus sem tempor orci, at finibus justo libero ut dui. Phasellus condimentum felis rutrum ex ultrices auctor. Maecenas dapibus magna ut eros ornare auctor. Aenean accumsan at ante quis bibendum. Duis nec nulla vel eros vestibulum commodo. Donec faucibus fringilla turpis sit amet vestibulum. Cras consequat mauris erat, non ornare tellus iaculis vel. Aenean ac ex mattis massa egestas aliquam. Morbi eleifend tempor metus, quis accumsan mauris aliquam eu.
+		Vestibulum ultricies dui nisl, non sodales velit mattis molestie. Nullam ac mattis sapien. Fusce orci odio, consectetur nec feugiat sed, ultricies sed ante. Aenean tincidunt dictum interdum. Suspendisse eu tortor nibh. Aliquam egestas, metus eget venenatis auctor, justo ante rhoncus diam, eu cursus augue quam ac ante. Aliquam scelerisque aliquet dolor sit amet ornare.
+		Etiam non sem sit amet magna tincidunt semper. Suspendisse ligula urna, suscipit vel pellentesque ac, porttitor sed erat. Vestibulum ornare quis sem in aliquam. Vivamus in consectetur orci. Integer ut semper velit, eu venenatis elit. Proin sit amet sem rutrum, consectetur urna posuere.
+		`,
+		SessionToken: "Another secret string",
+		Expiration:   time.Now().Unix(),
+	}
+
+	suite.store.setMyGOOS("linux")
+	err := suite.store.SaveRoleCredentials("bar", rc)
+	assert.NoError(t, err)
+	rc2 := RoleCredentials{}
+	err = suite.store.GetRoleCredentials("bar", &rc2)
+	assert.NoError(t, err)
+	assert.Equal(t, rc, rc2)
+	err = suite.store.DeleteRoleCredentials("bar")
+	assert.NoError(t, err)
```

Comment:
Occurs the same if you call it before my changes, but there's a thing I don't understand. This is the content of RoleCredentials after calling Delete  bar credentials in Linux version. 

![image](https://user-images.githubusercontent.com/1116133/160244350-4f4f7293-7b19-437a-a976-e9bc02305046.png)

The same occurs in windows, after calling delete you aren't saving the changes in keyring? or I'm missing something?
![image](https://user-images.githubusercontent.com/1116133/160244459-79c6e34b-4c14-49f5-be49-a0ed45359e7e.png)

Reply:
So two things:
 1. Sorry, I should of been more clear... I was asking you to write a unit test for that

 2. Uh, hey that's totally a bug and I should of had a unit test for that. :)  I've opened https://github.com/synfinatic/aws-sso-cli/issues/319 and will just address it all at once.  So you can just ignore my above request for a unit test since this is all on me.